### PR TITLE
Alane branch

### PR DESCRIPTION
**Pull request by bibang007**
model.js uploaded

### CODE REVIEW

File: models/models.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+const db = new Sequelize({
+  database: 'users_db',
+  dialect: 'postgres',
+  operatorsAliases: false,
+  define: { 
+    underscored: true
+  }
+});
+
+// define building model
```
Comment (imanipaul) [inline models/models.js]: should this be //define User model?

File: models/models.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+const User = db.define('user', {
+  name: {
+    type: Sequelize.VARCHAR
```
Comment (imanipaul) [inline models/models.js]: maybe the type should be Sequelize.STRING since it wont be that long

File: models/models.js [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+  },
+  email: {
+    type: Sequelize.VARCHAR,
+    unique: true
+  },
+  password_digest: {
+    type: Sequelize.STRING
```
Comment (imanipaul) [inline models/models.js]: should we add unique: true for the pasword_digest?

File: models/models.js [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+  }
+});
+
+// define category model
+
+const Category = db.define('category', {
+  name: {
+    type: Sequelize.VARCHAR
+  },
+  content: {
```
Comment (imanipaul) [inline models/models.js]: Category does not need a content column

File: models/models.js [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+const Thread = db.define('thread', {
+  title: {
+    type: Sequelize.VARCHAR
+  },
+  content: {
+    type: Sequelize.TEXT
+  }
+})
+
+// define thread model
```
Comment (imanipaul) [inline models/models.js]: Should be 'define Comment model'

File: models/models.js [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+
+
+User.hasMany(Thread, { onDelete: 'cascade' });
+User.hasMany(Comment, { onDelete: 'cascade' });
+Category.hasMany(Thread);
+Thread.belongsTo(Category);
+Thread.belongsTo(User);
+Comment.belongsTo(User);
+Comment.belongsTo(Thread);
+
```
Comment (imanipaul) [inline models/models.js:70]: Missing: Thread.hasMany(Comment)

**Review by imanipaul — COMMENTED** [review]
Looks good, needs just a few edits

**Comment by bibang007** [comment]
updated models.js file

**Review by imanipaul — APPROVED** [review]
Great, changes look good

**Pull request merged**